lib/components/standform/form.tsx

```diff
@@ -0,0 +1,166 @@
+import { Button } from "@/lib/components/ui/button";
+import { StandFormData, validate } from "@/lib/actions/stand-form";
+import { TabsContentForceMount } from "./force-mount-tab";
+
+export function EndgameContent() {
+	const [climbed, setClimbed] = useState(false);
+	const [parked, setParked] = useState(false);
+
+	useEffect(() => {
+		if (climbed) {
+			setParked(false);
+		}
+	}, [climbed]);
+
+	useEffect(() => {
+		if (parked) {
+			setClimbed(false);
+		}
+	}, [parked]);
+
+	return (
+		<div className="space-y-2">
+			<div className="flex items-center space-x-2">
+				<Checkbox
+					id="climbed"
+					className="size-6"
+					name="climbed"
+					checked={climbed}
+					onCheckedChange={() => setClimbed((curr) => !curr)}
+				/>
+				<Label htmlFor="climbed">Climbed?</Label>
+			</div>
+			<div className="flex items-center space-x-2">
+				<Checkbox
+					id="parked"
+					className="size-6"
+					name="parked"
+					checked={parked}
+					onCheckedChange={() => setParked((curr) => !curr)}
+				/>
+				<Label htmlFor="parked">Parked?</Label>
+			</div>
+			<div hidden={!climbed}>
+				<Label htmlFor="bots_on_chain">Bots Same Chain</Label>
+				<CounterInput
+					defaultValue={1}
+					required
+					min={1}
+					max={3}
+					name="bots_on_chain"
+				/>
+			</div>
+		</div>
+	);
+}
+
+export function StandForm({
+	handleSubmit,
+}: {
+	handleSubmit: (data: StandFormData) => Promise<unknown>;
+}) {
+	const [errors, setErrors] = useState([] as string[]);
+	const [activeTab, setActiveTab] = useState("auto");
+
+	const tabs = [
+		{
+			value: "auto",
+			content: <AutoTab />,
+			displayText: "Auto",
+		},
+		{
+			value: "teleop",
+			content: <TeleopTab />,
+			displayText: "Teleop",
+		},
+		{
+			value: "endgame",
+			content: <EndgameTab />,
+			displayText: "Endgame",
+		},
+		{
+			value: "misc",
+			content: <MiscTab />,
+			displayText: "Misc",
+		},
+	];
+
+	return (
+		<form
+			action={async (d) => {
+				// TODO: use useContext to get user info? or pass in w/ props
+				d.append("scouter", "0dc19558-ed9c-49c9-b1ba-9dca484c1954");
+				d.append("team_number", "1");
+				d.append("match_number", "1");
+				d.append("event_code", "2024test");
+
+				const res = validate(d);
+
+				if (!res.success) {
+					setErrors(res.errors);
+					return;
+				} else {
+					// TODO: error handling in submit function
+					handleSubmit(res.data!);
+					setErrors([]);
+				}
+			}}
+		>
+			<Card className="prose w-fit dark:prose-invert prose-headings:font-extrabold prose-h3:my-2 prose-h4:text-xl">
+				<CardHeader>
+					<CardTitle>Stand Form</CardTitle>
+				</CardHeader>
+				<CardContent className="px-6 pb-6">
+					<Tabs
+						value={activeTab}
+						onValueChange={setActiveTab}
+						className="max-w-[400px]"
+					>
+						<TabsList>
+							{tabs.map(({ value, displayText }) => (
+								<TabsTrigger key={value} value={value}>
+									{displayText}
+								</TabsTrigger>
+							))}
+						</TabsList>
```

Comment:
This is left-aligned right now—probably a bit tricky to get it to take up the full width. I'd recommend having it be center aligned.